src/components/auth0NavBar/LogoutButton.js

```diff
@@ -0,0 +1,14 @@
+import React from "react";
+import { useAuth0 } from "@auth0/auth0-react";
+
+const LogoutButton = () => {
+  const { logout, isAuthenticated } = useAuth0();
+
+  return isAuthenticated && (
+    <button className="btn btn-candy" onClick={() => logout({ returnTo: window.location.origin })}>
```

Comment:
Logs you out to the home page since we cannot wildcard pages I just redirect them back to `https:localhost:8000/`